src/s2wasm.h

```diff
@@ -491,8 +494,11 @@ class S2WasmBuilder {
       skipWhitespace();
       if (match(".p2align")) s = strchr(s, '\n');
       else if (match(".int32")) {
-        initializerFunctions.emplace_back(cleanFunction(getStr()));
-        assert(implementedFunctions.count(initializerFunctions.back()));
+        do {
```

Comment:
how about this (instead of what you have for 496-507)
```
    mustMatch(".int32"); // replaces the abort_on(...) below
    do {
      initializerFunctions.emplace_back(...);
      assert(...);
      skipWhitespace();
    } while (match(".int32"));
  } // end while (*s)
} // end function
```